src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -573,11 +548,10 @@ private List<Transaction> Reorder(List<Transaction> transactions)
                             node.DependsOn.Remove(parent);
                     }
 
-                    if (node.DependsOn.Count == 0)
-                    {
-                        result.Add(node.Transaction);
-                        dictionary.Remove(node.Hash);
-                    }
+                    if (node.DependsOn.Count != 0) continue;
```

Comment:
hmm weird way of writing but ok.

Reply:
I am (almost) always doing that to try and reduce nesting, if most people don't like I can revert

Reply:
its ok, if you dont object put `continue;` on the next line and we are good.

Reply:
ok